src/utils/misc.ts

```diff
@@ -31,3 +31,14 @@ export function toConditionalOrderParams({
     staticInput: staticInput.toString(),
   };
 }
+
+export function getAreConditionalOrderParamsEqual(
+  a: ConditionalOrderParams,
+  b: ConditionalOrderParams
+): boolean {
+  return (
+    a.handler === b.handler &&
+    a.salt === b.salt &&
+    a.staticInput === b.staticInput
+  );
```

Comment:
Do we care about case sensitivity?

Reply:
I don't think this is a case, but can't 100% be sure.
So, I would better to lowercase them